File: src/library/scala/runtime/java8/JFunction0$mcB$sp.java

/*
 * Copyright (C) 2012-2015 Typesafe Inc. <http://www.typesafe.com>
 */

package scala.runtime.java8;

@FunctionalInterface
public interface JFunction0$mcB$sp extends scala.Function0, java.io.Serializable {
    byte apply$mcB$sp();

    default Object apply() { return scala.runtime.BoxesRunTime.boxToByte(apply$mcB$sp()); }
}

File: src/library/scala/runtime/java8/JFunction0$mcB$sp.scala
/*
 * Scala (https://www.scala-lang.org)
 *
 * Copyright EPFL and Lightbend, Inc. dba Akka
 *
 * Licensed under Apache License 2.0
 * (http://www.apache.org/licenses/LICENSE-2.0).
 *
 * See the NOTICE file distributed with this work for
 * additional information regarding copyright ownership.
 */

package scala.runtime.java8

@FunctionalInterface trait JFunction0$mcB$sp extends Function0[Any] with Serializable {
  def apply$mcB$sp(): Byte
  override def apply(): Any = scala.runtime.BoxesRunTime.boxToByte(apply$mcB$sp())
}
